### PR TITLE
Add few annotations added by the kubelet

### PR DESCRIPTION
**Pull request by tengqm**
This PR adds some descriptions for the annotations added by the kubelet to Pods.

### CODE REVIEW

**Comment by kundan2707** [comment]
/retitle Add few annotations added by the kubelet

**Review by kumarankit999 — COMMENTED** [review]
Lgtm

**Review by kundan2707 — COMMENTED** [review]
/lgtm
thanks

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -434,6 +434,59 @@ The annotation `kubernetes.io/limit-ranger` records that resource defaults were
...
+Type: Annotation
+
+Example: `kubernetes.io/config.mirror: "df7cc47f8477b6b1226d7d23a904867b"`
+
+Used on: Pod
+
+For a static Pod created by the kubelet on a node, a {{< glossary_tooltip text="mirror Pod" term_id="mirror-pod" >}}
+is created on the API server. The kubelet adds an annotation to indicate that this Pod is
+actually a mirror Pod. The annotation value is copied from the [`kubernetes.io/config.hash`](#kubernetes-io-config-hash)
+annotation, which is usually the UID of the Pod.
```
Comment (kbhawkey) [inline content/en/docs/reference/labels-annotations-taints/_index.md:461]: nit: usually?
What else could it be?

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 2]
```diff
@@ -434,6 +434,59 @@ The annotation `kubernetes.io/limit-ranger` records that resource defaults were
...
+
+Type: Annotation
+
+Example: `kubernetes.io/config.source: "file"`
+
+Used on: Pod
+
+This annotation is added by the kubelet to indicate where the Pod comes from.
+For static Pods, the annoation value could be one of `"file"` or `"http"` depending
+on where the Pod manifest is located. For a Pod created on the API server and then
```
Comment (kbhawkey) [inline content/en/docs/reference/labels-annotations-taints/_index.md:476]: nit: The possible values are: `file`, `http`, or `api`? Do you need to double quote the values and surround in code blocks?
nit: Spelling error: annoation -> annotation

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 3]
```diff
@@ -434,6 +434,59 @@ The annotation `kubernetes.io/limit-ranger` records that resource defaults were
...
+
+### kubernetes.io/config.seen
+
+Type: Annotation
+
+Example: `kubernetes.io/config.seen: "2023-10-27T04:04:56.011314488Z"`
+
+Used on: Pod
+
+When the kubelet receives a Pod for the first time, it may add this annotation to
```
Comment (kbhawkey) [inline content/en/docs/reference/labels-annotations-taints/_index.md:487]: nit: Is `receives` the right word to use in this context?

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 2 above)
Comment (tengqm) [reply content/en/docs/reference/labels-annotations-taints/_index.md:476]: I was trying to emphasize that the value are strings. The double quotes can be removed.

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 3 above)
Comment (tengqm) [reply content/en/docs/reference/labels-annotations-taints/_index.md:487]: Alright, I'm changing it to "sees".

**Comment by kbhawkey** [comment]
Thanks @tengqm .
/lgtm
/approve

**Pull request merged**